common/zod/schema.ts

```diff
@@ -49,6 +49,8 @@ const BaseUserSchema = z.object({
   isForeignStudent: z.boolean().default(false),
   grade: GradeSchema,
 
+  wantToMatch: z.boolean(),
```

Comment:
```suggestion
  wantToMatch: z.boolean().default(true),
```
にしてweb/src/app/[locale]/registration/page.tsxのuseEffectを消すのはダメなのでしょうか？